src/mongocxx/include/mongocxx/v_noabi/mongocxx/bulk_write.hpp

```diff
@@ -58,6 +58,13 @@ class bulk_write {
     ///
     ~bulk_write();
 
+    ///
+    /// Checks if a bulk write operation is empty.
+    ///
+    /// @return A boolean that returns true if the bulk write operation is empty.
```

Comment:
```suggestion
    /// @return A boolean indicating if the bulk write operation is empty.
```

Suggest tweaking wording to avoid "boolean that returns".